specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json

```diff
@@ -630,6 +630,78 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/purge": {
+      "post": {
+        "description": "Purges data in an Log Analytics workspace by a set of user-defined filters.",
+        "operationId": "Workspace_Purge",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspacePurgeParameter"
+          }         
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted request for purging an Log Analytics workspace.",
+            "schema": {
+              "$ref": "#/definitions/WorkspacePurgeResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkspacePurge": {
+            "$ref": "./examples/WorkspacesPurge.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/operations/{purgeId}": {
```

Comment:
URL which will be used to track the status of the long running operation is obtained from the header.  You dont need to add this API here.

Reply:
Removed 👍